doc/userGuide.md

```diff
@@ -0,0 +1,182 @@
+# Running AdoptOpenJDK Tests
+
+Note: this document is currently being rewritten, if you have immediate test-related questions, please post them to the [AdoptOpenJDK testing Slack channel](https://adoptopenjdk.slack.com/messages/C5219G28G).
+
+Platform: x64_linux | x64_mac | s390x_linux | ppc64le_linux | aarch64_linux
+
+Java Version: SE80 | SE90 | SE100
+
+Prerequisites:
+	
+    perl 5.10.1 or above with JSON and Text::CSV, XML::Parser module installed
+    make 3.81 or above (recommend make 4.1 or above on windows)
+    wget 1.14 or above
+    ant 1.7.1 or above with ant-contrib.jar
+
+## Jenkins setup and running
+While you can run all the tests manually via the make targets on the command line, you may also run the tests in Jenkins. As part of the AdoptOpenJDK continuous integration (CI), AdoptOpenJDK runs test builds against the release and nightly SDK builds.
+
+You can easily set up your own Jenkins-based test builds using the AdoptOpenJDK openjdk-tests Jenkinsfile by:
+	
+	* Configure machines using playbooks at https://github.com/AdoptOpenJDK/openjdk-infrastructure/tree/master/ansible/playbooks
+	* Set up similar machine labels
+	* Tar sdk as *.tar.gz
+
+### Setting up a test build against SDK with URL :
+
+	* Create Pipeline test build job using Pipeline script from SCM :https://github.com/AdoptOpenJDK/openjdk-tests.git
+	* Build with Parameters
+		* Choose SDK_RESOURCE customized
+		* Setting CUSTOMIZED_SDK_URL to your customized SDK URL
+		* Setting all other parameters such as PLATFORM, JAVA_VERSION, etc.
+### Setting up a test build against user local SDK :
+	
+	* In plan
+### Setting up a test build against a SDK build
+	
+	* Create Pipeline test build job using Pipeline script from SCM :https://github.com/AdoptOpenJDK/openjdk-tests.git
+	* Configure your sdk build to trigger your created test build
+		* If using execute shell use Parameterized Trigger Plugin to passing in parameters including your sdk job name and job number
+		* If using pipeline update pipeline using following code : 
+
+```
+build job: 'test_build_name', parameters: [string(name: 'PLATFORM', value: '...'), string(name: 'JAVA_VERSION', value: '...'), ... string(name: 'UPSTREAM_JOB_NAME', value: "your sdk job name"), string(name: 'UPSTREAM_JOB_NUMBER', value: "your sdk job number")]
+```
+
+## Local setup and running (via make targets on the command line)
```

Comment:
Some of the information mentioned below is about how to use TestKitGen. I think we should redirect user to https://github.com/eclipse/openj9/blob/master/test/README.md, so we can avoid to maintain two files. For now, I think we can just add a link to openj9 test readme.